libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/resourcelists/ResourceComparatorGenerator.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I like this solution with this `ResourceComperatorGenerator` class.
We can easily enhance it by adding the other sw360 resource types.